packages/repeater/src/lib/RepeaterFactory.ts

```diff
@@ -86,30 +45,16 @@ export class RepeaterFactory {
       );
     }
 
-    const bus = container.resolve<EventBus>(EventBus);
-
-    await bus.init?.();
-
-    return bus;
-  }
-
-  private generateName(
-    namePrefix: string,
-    disableRandomNameGeneration: boolean = false
-  ) {
-    const normalizedPrefix = namePrefix?.trim();
-    const randomPostfix = disableRandomNameGeneration ? '' : `-${uuidv4()}`;
-    const name = `${normalizedPrefix}${randomPostfix}`;
-
-    if (name.length > this.MAX_NAME_LENGTH) {
-      const maxPrefixLength = this.MAX_NAME_LENGTH - randomPostfix.length;
-
-      throw new Error(
-        `Name prefix must be less than or equal to ${maxPrefixLength} characters.`
-      );
-    }
-
-    return name;
+    container.register<DefaultRepeaterServerOptions>(
+      DefaultRepeaterServerOptions,
+      {
+        useValue: {
+          uri: `${this.configuration.api}/workstations`,
+          token: this.configuration.credentials.token,
+          connectTimeout: 10000
+        }
+      }
+    );
```

Comment:
Why not use a factory provider? Why do you need to register it dynamically when the value is static?